tests/bumper-spec.js

```diff
@@ -216,7 +221,7 @@ describe('Bumper', () => {
       sandbox.stub(bumper.ci, 'commit').returns(Promise.resolve('committed'))
       sandbox.stub(bumper.ci, 'setupGitEnv').returns(Promise.resolve('set-up'))
 
-      return bumper._commitChanges().then((res) => {
+      return bumper._commitChanges({blackduck: 'blackduck'}).then((res) => {
```

Comment:
A better test, which would have caught the bug I mentioned earlier, would be to call it something other than `blackduck/`, since it's currently adding the hard-coded value of `blackduck/`, so something like
```js
return bumper._commitChanges({blackduck: 'my-blackduck-dir'}).then((res) => {
...
```

Or, better yet:
```js
return bumper._commitChanges({dependencyDir: 'my-dependency-dir'}).then((res) => {
```